performant-software/Annotation-Studio
# Fix spacing under banner and above user/document tab on anthology show view

## What does this PR do?
1. Fix spacing under banner and above user/document tab on anthology show view

## How to test?
1. Sign in and go to http://cove-staging.herokuapp.com/anthologies/sample-anthology
2. Check to see if there is enough space below the banner and above the tabs
